java-manta-client-unshaded/src/main/java/com/joyent/manta/config/BaseChainedConfigContext.java

```diff
@@ -435,83 +446,66 @@ public void overwriteWithContext(final ConfigContext context) {
 
             this.privateKeyContent = context.getPrivateKeyContent();
         }
-
```

Comment:
Please do not make large whitespace changes in the same PR as functional changes.

Reply:
I had to make those changes since they were generating a checkstyle error since we upgraded the dependency (```[ERROR]:Method body shouldn't exceed more than 150 lines```), so in order to ensure that the whitespace was consistent across the method, I made this change.

Reply:
I would either suppress the checkstyle check or break that method into smaller methods.

Reply:
Error Logs:
```
[ERROR] src/main/java/com/joyent/manta/config/BaseChainedConfigContext.java:[390,5] (sizes) MethodLength: Method length is 153 lines (max allowed is 150).
[ERROR] src/main/java/com/joyent/manta/config/SettableConfigContext.java:[296,5] (sizes) MethodLength: Method length is 153 lines (max allowed is 150).
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-checkstyle-plugin:2.17:check (checkstyle) on project java-manta-client-unshaded: You have 2 Checkstyle violations. -> [Help 1]
```